.github/workflows/test.yml

```diff
@@ -20,9 +20,9 @@ jobs:
           - target: aarch64-unknown-linux-gnu
             os: [self-hosted, linux, arm64]
           - target: x86_64-apple-darwin
```

Comment:
This workflow is somewhat broken... Or rather, misleading...

For example, the oldstable test is using `aarch64-apple-darwin` even when the `target` is set to `x86_64-apple-darwin` (See [the logs][1])

[1]: https://github.com/phylum-dev/birdcage/actions/runs/9845290932/job/27180501480?pr=78#step:5:22

I actually suspect that the same problem exists with the stable test, but it's not possible to tell from the logs on that one...

The simplest way to fix this is to remove the `target` value from this workflow entirely. And since it seems GitHub macos runners are all moving toward arm64, that means removing the `x86_64-apple-darwin` test entirely.

Reply:
Requested updates made.